execution_engine_testing/test_support/src/internal/wasm_test_builder.rs

```diff
@@ -896,6 +896,36 @@ where
         ret
     }
 
+    pub fn get_seigniorage_recipients_snapshot(&mut self) -> SeigniorageRecipientsSnapshot {
+        let correlation_id = CorrelationId::new();
+        let state_root_hash = self.get_post_state_hash();
+
+        let tracking_copy = self
+            .engine_state
+            .tracking_copy(state_root_hash)
+            .unwrap()
+            .unwrap();
+
+        let reader = tracking_copy.reader();
+
+        let era_ids = reader
+            .keys_with_prefix(correlation_id, &[KeyTag::EraValidators as u8])
+            .unwrap_or_default();
+
+        let mut ret = BTreeMap::new();
+
+        for era_id in era_ids.into_iter() {
```

Comment:
We want to apply the same windowing here that you do below in `get_seigniorage_recipients_snapshot`: we don't want to return the validator set for every era, just the current era and every era through the auction delay.

Reply:
Ahhh, actually @mpapierski and I dicussed that and decided that for the test_builder it can remain as it

Reply:
Okay fair.